src/_includes/toc-and-content.html

```diff
@@ -45,6 +45,6 @@ <h3 id="{{ subsection_item.url }}">{{ subsection_item.title }}</h3>
     <h4>{{ paragraph.title }}</h4>
     {% endif %}
     <p>{{ paragraph.text }}</p>
-    {% endfor %} {% endfor %} {% endfor %}
+    {% if paragraph.html %} {% include {{paragraph.html}} %}{% endif %} {% endfor %} {% endfor %} {% endfor %}
```

Comment:
I don't like how all the `{% endfor %}`s are all in one line, but that's how the formatter forces it to be.